src/Features/CSharp/Portable/CodeFixes/HideBase/HideBaseCodeFixProvider.RemoveNewKeywordAction.cs

```diff
@@ -0,0 +1,79 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.LanguageServices;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeFixes.HideBase
```

Comment:
odd namespace for this feature.

Reply:
Moved to a separate class in `Microsoft.CodeAnalysis.CSharp.CodeFixes.RemoveNewModifier`.